client/checkout/blocks/upe-deferred-intent-creation/payment-elements.js

```diff
@@ -62,8 +57,11 @@ const PaymentElements = ( { api, ...props } ) => {
 			>
 				<PaymentProcessor
 					api={ api }
-					errorMessage={ errorMessage }
+					errorMessage={ fingerprintErrorMessage }
```

Comment:
It looks like there is no need to have an additional `setState`. The `fingerprintErrorMessage` can just be forwarded to the `PaymentProcessor` component without additional re-rendering.